howsthewaterfront/src/components/SearchResult.js

```diff
@@ -1,7 +1,194 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import Search from "./Search";
+import CopyrightFooter from "./Footer";
+import toiletIcon from "../assets/icons8-toilet-50.png";
+import parkingIcon from "../assets/icons8-parking-60.png";
+import wheelchairIcon from "../assets/icons8-wheelchair-48.png";
+import strollerIcon from "../assets/icons8-stroller-50.png";
+import dogIcon from "../assets/icons8-dog-paw-64.png";
+import volleyIcon from "../assets/icons8-volleyball-player-50.png";
+import picnicIcon from "../assets/icons8-picnic-table-50.png";
+import surf_header from "../assets/beach_surf_header.jpg";
+import placeholder01 from "../assets/beach-placeholder01.png";
+import placeholder02 from "../assets/beach-placeholder02.png";
+import placeholder03 from "../assets/beach-placeholder03.png";
+import "../styles/search-result.css";
+
+const beachDummyData = {
+  beachName: "Marina del Rey Harbor",
+  address: "small-craft harbor, restaurants, hotels",
+  phoneNumber: "310-457-8143",
+  about:
+    "small-craft harbor, restaurants, hotels, waterfront parks, Visitor Center at 4701 Admiralty Way",
+  lat: "33.97",
+  lng: "Longitude",
+  region: "North California",
+  sunrise: "6:09 am",
+  sunset: "7:29 pm",
+  dayWeather: "34 - 45 F",
+  lowTide: "10:00 am 9:00 pm",
+  highTide: "5:00 am 6:00 pm",
+  windSpeed: "25 mph",
+  windDirection: "NE",
+  swellHeight: "2.0 m",
+  waterTemp: "40 F"
+  // ammeneties: []
```

Comment:
list of all amenities:

ID
DISTRICT
CountyNum
COUNTY
NameMobileWeb
LocationMobileWeb
DescriptionMobileWeb
PHONE_NMBR
FEE
PARKING
DSABLDACSS
RESTROOMS
VISTOR_CTR
DOG_FRIENDLY
PCNC_AREA
CAMPGROUND
SNDY_BEACH
DUNES
RKY_SHORE
BLUFF
STRS_BEACH
PTH_BEACH
BLFTP_TRLS
BLFTP_PRK
WLDLFE_VWG
TIDEPOOL
VOLLEYBALL
FISHING
BOATING
LIST_ORDER
GEOGR_AREA
C. Crescent City
LATITUDE
LONGITUDE
Photo_1
Photo_2
Photo_3
Photo_4
Bch_whlchr
BIKE_PATH
BT_FACIL_TYPE
REGION